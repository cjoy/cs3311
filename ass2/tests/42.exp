Allison Memari (3306815)
Suggestions for study in 11s2

Course   Title                 UOC Requirement
-------- -----                ---- -----------
COMP1927 Computing 2             6 Level 1 Core Courses
ENGG1000 Engineering Design      6 Level 1 Core Courses
MATH1231 Mathematics 1B          6 Level 1 Core Courses
MATH1241 Higher Mathematics 1    6 Level 1 Core Courses
SENG1031 Software Eng Worksho    6 Level 1 Core Courses
Free.... Free Electives (many    6 Free elect
