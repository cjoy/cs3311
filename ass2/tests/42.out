Allison Memari (3306815)
Suggestions for study in 11s2

167 168Course   Title                 UOC Requirement
-------- -----                ---- -----------
